### DISCUSSION TITLE
Where can I give feedback on this and stuff?

### DISCUSSION THREAD

**Discussion by Defiant-Firefly** [GDQuest/learn-gdscript#831] [Q&A]
Hi, if you can't tell, I've never used Github before so this may be an obvious one to most and I apologise if I'm doing this wrong in advance.
I'm going through the app thingo and I'm coming across multiple issues in the general way things are being explained and assumed and I'm not entirely sure where or if I can explain these things to hopefully help it improve. I see the 'ideas' tab but that seems to be for if you have ideas with a little bit more technical knowledge than I can give since I'm one of these people learning from zero.

**Comment by NathanLovato** [comment]
Hi. In general this is something you could do here.

Please note though have accumulated so much feedback on this, and we already have a plan for a complete remake from scratch in partnership with schools and teachers.

If you really want to take the time and write detailed feedback we would read and consider it, but please note that the state of the app does not mean that we think it's perfect. To really address fundamental pitfalls this thing needs a remake and this will take months of work. The remake will come out later this year.

**Reply by Defiant-Firefly** [reply]
Oh that's cool! I'm sure most points I would want to bring up would already be mentioned, but in the case of still wanting to give feedback when I get to the end of these lessons, where specifically would I put it? Like, this is the Q&A section, so would it go in the general section or ideas or somewhere else?

**Reply by NathanLovato** [reply]
Please feel free to use this very thread, it'd be great to have all your feedback in one place. You can reply at the bottom of the page each time, and you can just write something like this

## Lesson 2

- Feedback point 1
- Feedback point 2
- ...

Organized like this it'd be amazing to have.

Thanks for taking the time to give feedback!
